postgres/organisation_service.go

```diff
@@ -51,6 +51,20 @@ func (o *OrganisationService) AddOrganisation(
 	return convertOrganisation(organisation)
 }
 
+// Calls UpdateOrganisation query
+func (o *OrganisationService) UpdateOrganisation(
+	ctx context.Context,
+	organisationID core.OrganisationID,
+	name string,
+) error {
+	intID := int32(organisationID)
+	err := o.q.UpdateOrganisation(ctx, intID, name)
+	if err != nil {
+		return ConvertPgError(err)
+	}
+	return nil
```

Comment:
`ConvertPgError(nil) == nil`, dus je kan in principe ook gewoon dat teruggeven zonder de extra if 😛